deploy.sh

```diff
@@ -2,10 +2,11 @@
 
 PUBLIC_DATA_PATH=../covid-projections/public/data
 
-
 # Run State and County level models
 ./run_model.py state -o ${PUBLIC_DATA_PATH}
 ./run_model.py county -o ${PUBLIC_DATA_PATH}/county
 ./run_model.py county-summary -o ${PUBLIC_DATA_PATH}/county_summaries
 # Generate the latest state case summary data.
 ./run_data.py latest -o ${PUBLIC_DATA_PATH}/case_summary
+
+./deploy_dod_dataset.py -i ${PUBLIC_DATA_PATH} -o ${PUBLIC_DATA_PATH}
```

Comment:
can you output this to a folder in `covid-data-model` instead of the public data path for the website?